tt_metal/common/core_coord.h

```diff
@@ -66,19 +66,19 @@ inline CoreCoord get_core_coord_from_relative(const RelativeCoreCoord &in, const
 }
 
 struct CoreRange {
-    CoreCoord start;
-    CoreCoord end;
+    CoreCoord start_;
+    CoreCoord end_;
```

Comment:
I am not clear how renaming variables from proper start/end to start_/end_ help to address the iteration problem.
It makes code worse and it makes this PR unreasonable large.

Can you please revert the change to names?